### ISSUE TITLE
Bring in the “Page Patterns” that .com show when creating a new page.

### ISSUE THREAD

**Issue by mikestottuk** [Automattic/jetpack#31990]
When creating a page on WP.com there's a number of patterns shown as an option, including Newsletter patterns (page suggestions).

We should consider having these shown to Jetpack users. I think this would involve bringing the Page Pattern Modal into a shared package between JP and Calypso 

`wp-calypso/packages/page-pattern-modal/src/components/page-pattern-modal.tsx`

![Image](https://github.com/Automattic/jetpack/assets/4077604/9f092b93-55de-44d7-aa99-a61ca114c873)

**Comment by dsas** [comment]
Dunno if this makes it easier, but it's exported as `@automattic/page-pattern-modal` we consume it in jetpack-mu-wpcom here: https://github.com/Automattic/jetpack/tree/trunk/projects/packages/jetpack-mu-wpcom/src/features/starter-page-templates

**Comment by richtabor** [comment]
I don't think we should do this for Jetpack. It's more likely to conflict with core/existing theme patterns than add value. 

Cool to close?

**Comment by ryelle** [comment]
It seems like this already exists on Atomic sites, but not self-hosted.

If I try to add a new page on a site with a classic theme I see it:

<img width="1670" alt="Image" src="https://github.com/user-attachments/assets/efe5095f-c06b-40cc-89ed-26f05ae8216c" />

And if I'm on a block theme, I see it for a second or so before the core pattern modal loads in.

https://github.com/user-attachments/assets/f86c1be3-06d1-414b-92ff-f7500465129b

If I'm on a simple site, I see the wpcom modal, but upon closing it I can select patterns from the core inserter. I'm not sure if that's a GB version difference or what (oh, probably, seems like it was disabled then restored).

https://github.com/user-attachments/assets/d2e2344c-bd9b-43c8-a051-fe114b145411

These differences all seem strange, I think if possible it would be better to get the wpcom-provided patterns into the core screens — they're already loaded as patterns, so it would "just" require adding the `post-content` block type meta, that would trigger core to filter it into the starter content category here:

<img width="624" alt="Image" src="https://github.com/user-attachments/assets/6dbc06b9-e4f6-4e88-90d7-c63ce6d8e67a" />

The drawback is that the wpcom page has filtering for the starter patterns, while the core modal does not, and the core modal can be turned off (but that's a benefit, IMO). I'm also not sure if adding patterns as starter content would be enough to get the modal to appear on classic themes.

**Comment by ryelle** [comment]
I just saw that this screen has been removed from wp.com in favor of the core behavior (see https://github.com/Automattic/jetpack/pull/41479, https://github.com/Automattic/wp-calypso/pull/99178), so I'm going to close this issue.
